### ISSUE TITLE
When a spy or stub has not been called, the error should be grammatically correct

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#520]
* Cypress Version: 0.19.2

### Are you requesting a feature or reporting a bug?
Bug

### Current behavior:
With this test code:

```coffeescript
it "stubs", ->
  obj = {
    foo: ->
  }

  cy.stub(obj, "foo")

  expect(obj.foo).to.be.calledWith("bar")
```

We get this error message:

![screen shot 2017-05-30 at 6 00 23 pm](https://cloud.githubusercontent.com/assets/1268976/26606964/163f9fde-4562-11e7-82d0-faebeac596e2.png)

### Expected behavior:

The error should say:

```
Expected foo to have been called with arguments "bar".

foo was never called.
```

**Comment by chrisbreiding** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
